application/testing/CMakeLists.txt

```diff
@@ -176,6 +176,8 @@ f3d_test(NAME TestLightIntensityDarker DATA cow.vtp ARGS --light-intensity=0.2 D
 f3d_test(NAME TestLightIntensityBrighterFullScene DATA WaterBottle.glb ARGS --light-intensity=5.0 THRESHOLD 60 DEFAULT_LIGHTS)
 f3d_test(NAME TestLightIntensityDarkerFullScene DATA WaterBottle.glb ARGS --light-intensity=0.2 DEFAULT_LIGHTS)
 f3d_test(NAME TestUTF8 DATA "(ノಠ益ಠ )ノ.vtp" DEFAULT_LIGHTS)
+f3d_test(NAME TestFilenameCommas DATA a,b,c.d REGEXP "a,b,c\.d\" does not exist")
+f3d_test(NAME TestFilenameSpaces DATA "a b c.d" REGEXP "a b c\.d\" does not exist")
```

Comment:
I would be more confortable with an actual test that open a file named like this. maybe put spaces and commas in a single file ?